tex/chapters/07-ApplyingModel.tex

```diff
@@ -0,0 +1,197 @@
+%!TEX root = ../main.tex
+
```

Comment:
You should also throw the data set you use to train the model somewhere (the final one with the 30 or so strategies and the ratios) somewhere. :+1: